triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/dao/DbTriggerDao.scala

```diff
@@ -174,6 +194,9 @@ final class DbTriggerDao private (dataSource: DataSource with Closeable, xa: Con
   override def addRunningTrigger(t: RunningTrigger): Either[String, Unit] =
     run(insertRunningTrigger(t))
 
+  override def getRunningTrigger(triggerInstance: UUID): Either[String, Option[RunningTrigger]] =
+    run(queryRunningTrigger(triggerInstance))
```

Comment:
Note, this part is not tested, yet. This depends on https://github.com/digital-asset/daml/issues/7720.